infra/charts/feast-feature-server/templates/deployment.yaml

```diff
@@ -48,11 +49,20 @@ spec:
             - "feast"
             - "serve_registry"
             {{- else }}
+            # Add the metric check if the metrics are enabled
+            # {{- if .Values.metrics }}
+            # - "feast"
+            # - "serve"
+            # - "--metrics"
+            # - "-h"
+            # - "0.0.0.0"
+            # {{- else }}
```

Comment:
shouldn't this block of configs be uncommented and enabled only when metrics are configured in the chart?

Reply:
these chart changes should all be fully vetted in a cluster to ensure it continues to work as expected with or without metrics configured.